src/scripts/transformRecordedTests/index.ts

```diff
@@ -3,11 +3,12 @@ import { identity } from "./transformations/identity";
 import { upgrade } from "./transformations/upgrade";
 import { transformFile } from "./transformFile";
 import { FixtureTransformation } from "./types";
+import { upgradeThatMarks } from "./upgradeThatMarks";
 
 const AVAILABLE_TRANSFORMATIONS: Record<string, FixtureTransformation> = {
   upgrade,
   autoFormat: identity,
-  // custom: MY_CUSTOM_TRANSFORMER,
+  custom: upgradeThatMarks,
```

Comment:
we'll revert this file before merging; it just imports the transformation I used on test cases